source/js/image-test-yalong.js

```diff
@@ -0,0 +1,28 @@
+const file = document.getElementById('recipeImage');
+console.log(file)
+
+file.addEventListener('change', e=>{
+    const formdata = new FormData();
+    formdata.append('image', e.target.files[0]);
+    console.log(file);
+    fetch('https://api.imgur.com/3/image/', {
+        method:'post',
+        headers:{
+            Authorization:'Client-ID 1b99956c57a5642'
+        },
+        body:formdata
+    }).then(data=>data.json()).then(data=>{
+        console.log(data)
+        let divImg = document.getElementById('img-spot');
```

Comment:
Codacy has a fix for the issue: ['divImg' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-fa21-group7/cse110-fa21-group7/pullRequest?prid=8427187)
```suggestion
        const divImg = document.getElementById('img-spot');
```